nextcloud_scan.py

```diff
@@ -48,12 +47,13 @@ def requestResult(self):
 
     def doScan(self):
         result = self.requestResult()
+
+        # Zeitzone extrahieren und ignorieren, da strptime in Python 3 keine %Z mehr unterstützt
         lastScanTimestamp = datetime.datetime.strptime(
-            result['scannedAt']['date']
-                + ' '
-                + result['scannedAt']['timezone'],
-            '%Y-%m-%d %H:%M:%S.%f %Z'
+            result['scannedAt']['date'],
+            '%Y-%m-%d %H:%M:%S.%f'
```

Comment:
I don't think you should completely ignore the time zone that scan.nextcloud.com returns. It *should* always return UTC, but since they've chosen to return the time zone in the API, we should handle it using `zoneinfo.ZoneInfo`.